pci/src/vfio.rs

```diff
@@ -1662,8 +1662,12 @@ impl VfioPciDevice {
                                 user_memory_region.host_addr,
                             )
                             .map_err(|e| {
-                                VfioPciError::DmaMap(e, self.device_path.clone(), self.bdf)
-                            })?;
+                                error!(
```

Comment:
Hhm - I guess this is correct - in terms of only being required for P2P (replicating the code archaeology you did.) Please can you add a comment at line 1657 saying that this code only applies for P2P support.